File: geekbrains/homework2/app/src/main/java/ru/moondi/homework2/MainActivity.java
package ru.moondi.homework2;

import androidx.appcompat.app.AppCompatActivity;


import android.os.Bundle;
import android.view.View;
import android.widget.Button;
import android.widget.TextView;

public class MainActivity extends AppCompatActivity {
    TextView text;
    int buffer;
    char operation;
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        buffer = 0;
        operation = '1';
        text = findViewById(R.id.textDisplay);
        Button clear = findViewById(R.id.button_clear);
        clear.setOnClickListener(View ->{
            text.setText("");
            buffer = 0;
        });
        Button equal = findViewById(R.id.button_equally);
        equal.setOnClickListener(View -> {
        switch (operation){
            case '+':
                buffer += Integer.parseInt(text.getText().toString());
                text.setText(String.valueOf(buffer));
                break;
            case '-':
                buffer -= Integer.parseInt(text.getText().toString());
                text.setText(String.valueOf(buffer));
                break;
            case '/':
                buffer /= Integer.parseInt(text.getText().toString());
                text.setText(String.valueOf(buffer));
                break;
            case '*':
                buffer *= Integer.parseInt(text.getText().toString());
                text.setText(String.valueOf(buffer));
                break;
        }
        });
    }
    public void intoNumber(View view){
        Button b = (Button) view;
        text.setText(text.getText().toString() + b.getText().toString());

    }
    public void calculate(View view){
        Button b = (Button) view;
    buffer = Integer.parseInt(text.getText().toString());
    operation = b.getText().charAt(0);
    text.setText("");
    }
}

File: geekbrains/01/app/src/main/java/ru/moondi/lesson1/MainActivity.kt
package ru.moondi.lesson1

import android.os.Bundle
import android.widget.Button
import android.widget.EditText
import android.widget.TextView
import androidx.appcompat.app.AppCompatActivity


class MainActivity : AppCompatActivity() {
    private var arg1: Int? = null
    private var arg2: Int? = null

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        val button = findViewById<Button>(R.id.button)
        val editText1 = findViewById<EditText>(R.id.editText1)
        val editText2 = findViewById<EditText>(R.id.editText2)
        val textView = findViewById<TextView>(R.id.textView)
        button.setOnClickListener {
            try {
                arg1 = Integer.valueOf(editText1.text.toString())
                arg2 = Integer.valueOf(editText2.text.toString())
                if (arg1 === arg2) {
                    textView.text = "Равно!"
                } else {
                    textView.text = "не Равно!"
                }
            } catch (e: Exception) {
                textView.setText("Введите число!");
            }
        }
    }
}
